jsonapi/fixtures_test.go

```diff
@@ -123,37 +153,65 @@ type ErrorIDPost struct {
 	Error error
 }
 
-func (s ErrorIDPost) GetID() string {
-	return ""
+func (s ErrorIDPost) GetID() Identifier {
+	return Identifier{ID: "", LID: "", Name: "error-id-posts"}
 }
 
-func (s *ErrorIDPost) SetID(ID string) error {
+func (s *ErrorIDPost) SetID(ID Identifier) error {
 	return s.Error
 }
 
 type Post struct {
 	ID            int           `json:"-"`
+	LID           int           `json:"-"`
 	Title         string        `json:"title"`
 	Comments      []Comment     `json:"-"`
 	CommentsIDs   []int         `json:"-"`
+	CommentsLIDs  []int         `json:"-"`
 	CommentsEmpty bool          `json:"-"`
 	Author        *User         `json:"-"`
 	AuthorID      sql.NullInt64 `json:"-"`
+	AuthorLID     sql.NullInt64 `json:"-"`
 	AuthorEmpty   bool          `json:"-"`
 }
 
-func (c Post) GetID() string {
-	return fmt.Sprintf("%d", c.ID)
+func (c Post) GetID() Identifier {
+	id := Identifier{ID: fmt.Sprintf("%d", c.ID), Name: "posts"}
```

Comment:
No need to define Name, since GetName was not implemented before

Reply:
removed